vllm/sequence.py

```diff
@@ -364,7 +364,7 @@ class SequenceGroupState:
     """Mutable state tied to a specific sequence group"""
 
     # torch.Generator used in seeded sampling
-    generator: Optional = None
+    generator: Optional[torch.Generator] = None
```

Comment:
potentially not imported

Reply:
I think it is fixed with the other PR that's merged!